Grids: Fix filterRow and filter(filterExpr) docissues (#2366)
DevExpress/devextreme-documentation: api-reference/10 UI Components/GridBase/1 Configuration/filterRow/showOperationChooser.md

> Specifies whether icons that open the filter lists are visible.

Reviewer: ```suggestion
Specifies whether or not icons that open the filter lists are visible.
```

---

> [note] If certain columns do not display the filter icons, specify the columns' [dataType]({basewidgetpath}/Configuration/columns/#dataType) and make sure that you did not disable their [allowFiltering]({basewidgetpath}/Configuration/columns/#allowFiltering) property.

Reviewer: ```suggestion
[note] If columns do not display filter icons, specify the columns' [dataType]({basewidgetpath}/Configuration/columns/#dataType) and ensure that the columns' [allowFiltering]({basewidgetpath}/Configuration/columns/#allowFiltering) property is not disabled. This will allow filter icons to be  displayed.
```
